charts/karmada/templates/_karmada_apiservice.tpl

```diff
@@ -57,4 +57,60 @@ spec:
   type: ExternalName
   externalName: {{ $name }}-search.{{ include "karmada.namespace" . }}.svc.{{ .Values.clusterDomain }}
 {{- end }}
-{{- end -}}
+{{- if has "metricsAdapter" .Values.components }}
+---
+apiVersion: apiregistration.k8s.io/v1
+kind: APIService
+metadata:
+  name: v1beta1.metrics.k8s.io
+  labels:
+    app: {{ $name }}-metrics-adapter
+    apiserver: "true"
+spec:
+  insecureSkipTLSVerify: true
```

Comment:
Hello, thank you for your important commit! 

While, we are trying to eliminate `insecureSkipTLSVerify=true` in karmada, detail in #4024

Can you refer to [karmada-metrics-adapter-apiservice.yaml](https://github.com/karmada-io/karmada/blob/master/artifacts/deploy/karmada-metrics-adapter-apiservice.yaml) and #4033 to remove `insecureSkipTLSVerify`?

Besides, could you please add steps of "how to verify your code" in this PR?